docs/additional-features/stress-testing.md

```diff
@@ -8,24 +8,46 @@ title: Network stress testing
 This guide assumes that:
 
 - You have a working Polygon-SDK network up and running
```

Comment:
`Polygon SDK` is more proper to be used?

Reply:
Fixed 👍 

74f7726